handbook/engineering/code-insights/index.md

```diff
@@ -0,0 +1,34 @@
+# Code insights team (coming soon)
```

Comment:
```suggestion
# Code insights team
```

I suggest we take the coming soon away, it doesn't add any value.

Reply:
I agree